test/unit/collection-view.spec.js

```diff
@@ -28,6 +28,34 @@ describe('collection view', function() {
   // Collection View Specs
   // ---------------------
 
+  describe('when a collection view is DOM', function() {
+    beforeEach(function() {
+      this.$fixtureEl = $('<div id="fixture-collectionview"></div>');
```

Comment:
it can also just be `new this.ChildView({ el: '#el1' });`  and I do think `addChildView` would work work.